frontend/src/components/module-editor/EditableContentBlock.tsx

```diff
@@ -54,12 +62,22 @@ const EditableContentBlock = ({
             {SelectContentBlock(block)}
             <EditContentOptionsMenu
               isVisible={isHovered}
-              onEditClick={() => {}}
+              onEditClick={() => {
+                onOpen();
+              }}
```

Comment:
Can you pass `onOpen` directly here? i.e.,
```suggestion
              onEditClick={onOpen}
```